models/v1beta3/academy/academy_helper.go

```
@@ -0,0 +1,79 @@
+// This is not autogenerated.
```

golangci-lint (2.12.2)

[error] ST1000: package comment should be of the form "Package category..."

(staticcheck)